CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [1.4.1] - 2022-09-07
+
+### Added
+- Support to node reattestation to certain Node Attestors (#3031)
+- Server now adds unique identifier to SVIDs subject (#3367)
+- Agents are shutting down when banned (#3308)
+
+### Fixed
+- Vault Upstream Authority to prevent adding duplicate certificates (#3376)
```

Comment:
I've made a suggestion to a change. I'm not sure we need to include this in the CHANGELOG though. There really isn't any user impact. This was more of a cleanliness change from what I understand. The X509CAChain is not stored in the bundle. The only impact from what I understand is the contents of the journal... which users won't notice....

What do you think?

```suggestion
- Vault Upstream Authority no longer adds duplicate certificates to the CA chain using Vault 1.11+ (#3376)
```

Reply:
mm I was not sure about adding it either...
I'm ok with removing it from changelog